services/utils/content.js

```diff
@@ -1,13 +1,39 @@
-const importItemByContentType = (id, item) => {
+const importItemByContentType = async (id, item) => {
+  // Get the model definition - Maybe there's a more elegant way?
+  const modelName = id.split('.').reverse()[0];
+  const {attributes} = strapi.models[modelName];
+  // Get which fields are unique indexes
+  const uniqueIndexes = Object.keys(attributes).filter(
+    (aName) => attributes[aName].index && attributes[aName].unique
```

Comment:
If a field is configured to be `unique`, there is only `unique: true`, how to make `index` property to appear and set to `true`?
Here's my simple test content type, I printed out the `attributes`
```json
{
  title: { type: 'string', unique: true },
  content: { type: 'richtext' }
}
```

Reply:
@lazurey I'm still debating on how to determine if a field should be used or not for uniquenes. Maybe just check "unique"...

Reply:
Understand, I was also wondering whether should use `id` to determine whether to update or create. But in the end I chose the lazy way.
I didn't test if there're two unique fields what will happen. If in the scenario, what is the expected behaviour?